lib/common/tests/xml/pcmk__xe_copy_attrs_test.c

```diff
@@ -14,49 +14,51 @@
 #include <glib.h>
 
 static void
-target_is_NULL(void **state)
+null_target(void **state)
 {
-    xmlNode *test_xml_1 = pcmk__xe_create(NULL, "test_xml_1");
-    xmlNode *test_xml_2 = NULL;
+    // This test dumps core via CRM_CHECK()
+    xmlNode *src = pcmk__xe_create(NULL, "test");
+    xmlNode *target = NULL;
 
-    pcmk__xe_set_props(test_xml_1, "test_prop", "test_value", NULL);
+    crm_xml_add(src, "attr", "value");
+    pcmk__xe_copy_attrs(target, src, true, false);
```

Comment:
overkill, but may as well test various combinations of bool args

Reply:
I'm redoing this with flags, and testing all combinations of flags is impractical in general. There will only be three for now but still.